draft-ietf-quic-datagram.md

```diff
@@ -57,8 +57,9 @@ Discussion of this work is encouraged to happen on the QUIC IETF mailing list
 
 The QUIC Transport Protocol {{!RFC9000}} provides a secure,
 multiplexed connection for transmitting reliable streams of application data.
-Reliability within QUIC is performed on a per-stream basis, so some frame
-types are not eligible for retransmission.
+QUIC uses various frame types to transmit data within packets, and each frame
+type defines whether or not the data it contains will be retransmitted.
+Streams of reliable application data are sent using STREAM frames.
```

Comment:
change look fine but is pretty duplicative of L74-75. Maybe remove that text?

Reply:
I agree with Lucas, the new text makes L74-75 redundant so I'd remove L74-75

Reply:
Sounds good, removed those redundant lines.